posthog/migrations/0094_auto_20201028_1208.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.0.7 on 2020-10-28 12:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0093_remove_user_is_superuser"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="dashboarditem",
+            name="description",
+            field=models.CharField(blank=True, max_length=400, null=True),
+        ),
+        migrations.AddField(model_name="dashboarditem", name="is_sample", field=models.BooleanField(default=False),),
+    ]
```

Comment:
Comment on the file overall: I'd change the name from `auto_{datetime}` to something more meaningful like `sample_dashboard_items` (also in `latest_migrations.manifest`)